### PR TITLE
Script to create new profiles on MongoDB

### PR DESCRIPTION
**Pull request by damianmarti**
Run _node scripts/createNewProfiles.js_ from packages/server to save on MongoDB all the new tokenIds.

### CODE REVIEW

File: packages/server/scripts/createNewProfiles.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { updateProfiles } = require('../services/profiles')
+const mongoose = require('mongoose')
+const config = require('config')
+const { dbUrl } = config.get('API_CONFIG.api')
+
+const createNewProfiles = async () => {
+  try {
+    await mongoose.connect(dbUrl)
+    await updateProfiles()
+    mongoose.disconnect()
```
Comment (QEDK) [inline packages/server/scripts/createNewProfiles.js:10]: Should be awaited

File: packages/server/scripts/createNewProfiles.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  } catch (err) {
+    throw Error(err)
```
Comment (QEDK) [inline packages/server/scripts/createNewProfiles.js:12]: This is equivalent to not catching the error, not needed then.

File: packages/server/services/profiles.js [hunk 3]
```diff
@@ -1,15 +1,36 @@
 /* eslint-disable */ // WIP
 const { dRecruitContract } = require('../helpers/contract')
 const { Profile } = require('../models')
+const { getDidFromTokenURI, selfIdCore } = require('../helpers/ceramic')
+const DEBUG = process.env.DEBUG
 
 const updateProfiles = async () => {
-  const lastTokenIdDb = await Profile.find({}, { _id: 0, tokenId: 1 }, { $sort: { tokenId: 1 }, limit: 1 })
+  const lastProfile = await Profile.findOne({}, { _id: 0, tokenId: 1 }, { sort: { tokenId: -1 } })
+  let lastTokenIdDb = 0;
```
Comment (QEDK) [inline packages/server/services/profiles.js:9]: you can use a nullable here, `lastProfile?.tokenId ?? 0`

File: packages/server/services/profiles.js [hunk 4]
```diff
@@ -1,15 +1,36 @@
...
+  if (lastProfile) {
+    lastTokenIdDb = lastProfile.tokenId
+  }
+  if (DEBUG) console.log("lastTokenIdDb: ", lastTokenIdDb);
```
Comment (QEDK) [inline packages/server/services/profiles.js:13]: Not following `standard` codestyle. Run `npx standard --fix .`

**Review by QEDK — CHANGES_REQUESTED** [review]

**Comment by damianmarti** [comment]
@QEDK changes commited!

**Review by QEDK — APPROVED** [review]

**Pull request merged**